test/unused-attributes.t

```diff
@@ -11,8 +11,9 @@
   File "x.ml", line 4, characters 5-14:
   4 |    [@bs.string]
            ^^^^^^^^^
-  Error (warning 101 [unused-bs-attributes]): Unused attribute: bs.string
+  Alert unused-bs-attributes: Unused attribute [@bs.string]
   This means such annotation is not annotated properly.
-  for example, some annotations is only meaningful in externals
+  For example, some annotations are only meaningful in externals
   
-  [2]
+  // Generated by Melange
+  /* This output is empty. Its source's type definitions, externals and/or unused code got optimized away. */
```

Comment:
It is a bit strange to include this in the error output. I understand it is caused by the hack to preserve `melc` behavior to avoid adding `-ppx melppx` to dune?